lib/item_marketplace/presenters/items_index_presenter.rb

```diff
@@ -5,4 +5,8 @@ class ItemsIndexPresenter
   def initialize(items)
     @items = items
   end
+
+  def available_items
+    @items.select {|item| item.available == true}
```

Comment:
Also, you'll probably want to pass in available items to the presenter. Unless the presenter will be querying based on any future pagination params. For example, if the controller passes `Item.limit(10)`, and the first 10 items are not available, you'll have 0 items to show. The controller should pass `Item.available.limit(10)`